database/sqlite/sqlite_health.c

```diff
@@ -1282,16 +1216,17 @@ int alert_hash_and_store_config(
     return 1;
 }
 
-#define SQL_SELECT_HEALTH_LAST_EXECUTED_EVENT "SELECT hld.new_status FROM health_log hl, health_log_detail hld WHERE hl.alarm_id = %u AND hld.unique_id != %u AND hld.flags & %u AND hl.host_id = @host_id and hl.health_log_id = hld.health_log_id ORDER BY hld.unique_id DESC LIMIT 1;"
+#define SQL_SELECT_HEALTH_LAST_EXECUTED_EVENT                                                                          \
+    "SELECT hld.new_status FROM health_log hl, health_log_detail hld "                                                 \
+    "WHERE hl.host_id = @host_id AND hl.alarm_id = @alarm_id AND hld.unique_id != @unique_id AND hld.flags & @flags "  \
+    "AND hl.health_log_id = hld.health_log_id ORDER BY hld.unique_id DESC LIMIT 1;"
```

Comment:
Considering we are still using `unique_id` as Unix epoch, I agree 100% that we need to take a look on them in `DESC` order, but we need to bring another change for this query to make more sense. We are asking in `DESC` for an index created in [ASC](https://github.com/netdata/netdata/blob/3c78c045bb0ea7225b35df4b6a83e671fc68aed1/database/sqlite/sqlite_functions.c#L58) order. Please, modify the index.